### PR TITLE
Added keyValueIterator() example, variable name cleanup

### PR DESCRIPTION
**Pull request by cambiata**

### CODE REVIEW

File: assets/content/cookbook/Beginner/arrays.md [hunk 1]
```diff
@@ -77,152 +86,166 @@ That means these are your options:
...
+    trace('$index : ${items[index]}');
 }
 ```
 
-### Operations
+As an alternative, you can use the `keyValueIterator()` method to get indexes and values:
+
+```haxe
+var var items = ["a", "b", "c"];
+for (index => item in items.keyValueIterator()) {
```
Comment (Gama11) [inline assets/content/cookbook/Beginner/arrays.md:132]: You don't need to (and shouldn't) call `keyValueIterator()` explicitly. Simply use `for (index => item in items)`.

File: assets/content/cookbook/Beginner/arrays.md [hunk 2]
```diff
@@ -77,152 +86,171 @@ That means these are your options:
...
+// c
+```
 
-The array defines an iterator, and its elements can therefore be iterated over.
+Using the `index => item` notation in the for loop is the simplest way to get the current item as well as its index:
 
 ```haxe
-for (item in arrayOfStrings) {
-    // do something
+var var items = ["a", "b", "c"];
```
Comment (Gama11) [inline assets/content/cookbook/Beginner/arrays.md:126]: `var var` :)

File: assets/content/cookbook/Beginner/arrays.md (hunk 1 above)
Comment (cambiata) [reply assets/content/cookbook/Beginner/arrays.md]: Ah, thanks - didn't know that!
Updated the example.

File: assets/content/cookbook/Beginner/arrays.md (hunk 2 above)
Comment (cambiata) [reply assets/content/cookbook/Beginner/arrays.md]: Live code review - that's great! :-)
Fixed.

**Comment by markknol** [comment]
Those are nice additions/changes, thanks for that!

**Pull request merged**